### DISCUSSION TITLE
Fluxograma

### DISCUSSION THREAD

**Discussion by mikaelmonteirodev** [EasyRoomCompany/Guardian#84] [General]
Segue um fluxograma esquematizando as funcionalidades da aplicação de acordo com as regras de negócio.

Clique neste link para abrir em PDF com mais qualidade: [EasyRoom.pdf](https://github.com/HackathonTeam05/Guardian/files/11547513/EasyRoom.pdf)

![Imagem ilustrativa](https://github.com/HackathonTeam05/Guardian/assets/112489130/e85404ee-ed67-460e-b479-5ffe3f920750)

**Comment by kevinDsousa** [comment]
Muito bom @mikaelmonteirodev nosso fluxograma para nos dar um norte em como implementar os requisitos

**Comment by kevinDsousa** [comment]
@mikaelmonteirodev Preciso que leve este documento para proxima reuniao para decidirmos como será feita a proxima sprint, precisamos refatorar alguns codigos entao vamos por partes

**Reply by mikaelmonteirodev** [reply]
Tudo bem. 👍

**Reply by kevinDsousa** [reply]
![image](https://github.com/EasyRoomCompany/Guardian/assets/55944691/129a82da-e411-4c99-b846-d8f1a2e3cd5f)

Análisando aqui com calma o documento, acredito que antes de iniciar de fato no dashboard, o usuário precisa ser checado se ele tem é administrador ou não, caso seja ir pro dash, caso não ir pra tela de agendamento.

**Reply by mikaelmonteirodev** [reply]
Exato!

**Comment by mikaelmonteirodev** [comment]
![image](https://github.com/EasyRoomCompany/Guardian/assets/112489130/c00cc4d9-2778-4b80-8af2-c736fd2643fe)

[Aqui o PDF completo do Fluxograma](https://github.com/EasyRoomCompany/Guardian/files/11990855/Fluxograma.pdf)
